orchagent/portsorch.cpp

```diff
@@ -297,6 +297,52 @@ static bool isValidPortTypeForLagMember(const Port& port)
     return (port.m_type == Port::Type::PHY || port.m_type == Port::Type::SYSTEM);
 }
 
+static std::string saiAttrToString(sai_port_serdes_attr_t attr)
+{
+    switch (attr)
+    {
+    case sai_port_serdes_attr_t::SAI_PORT_SERDES_ATTR_START:
```

Comment:
do we need SAI_PORT_SERDES_ATTR_START? it is not a real SAI attribute

Reply:
Didn't know that. I'm removing it from the switch statement.